dask/delayed.py

```diff
@@ -6,6 +6,7 @@
 from dataclasses import fields, is_dataclass
 
 from tlz import concat, curry, merge, unique
+from typing import TypeVar
```

Comment:
There's a small linting issue with the location of this `import` (see https://docs.dask.org/en/latest/develop.html#code-formatting)